src/Modules/UngradedModules/Presentation.jl

```diff
@@ -28,7 +28,7 @@ function presentation(SQ::SubquoModule)
     end
   else
     if is_graded(SQ)
-      s, _ = kernel(graded_map(ambient_free_module(SQ), gens(SQ.sum)))
+      s, _ = kernel(graded_map(ambient_free_module(SQ), filter(!iszero, gens(SQ.sum))))
```

Comment:
@jankoboehm : There is an issue when calling `graded_map` with any zero in the second argument. The error could be fixed here by adding the `filter`, but I am not 100% sure this is legitimate. Either way, the rest of the code for `presentation` should eventually be checked as to whether it suffers from similar issues. But I found it too hard to read to actually understand what's going on and simply adding `filter` to the other calls to `graded_map` made things worse. So I only did this one change for the moment.